api-gateway/src/main/resources/application-internal.yml

```diff
@@ -0,0 +1,50 @@
+gateway:
```

Comment:
looks like the old version of application-internal.yml was committed

Reply:
yes, fixed it